.gitignore

```diff
@@ -41,6 +41,7 @@ playground.xcworkspace
 # Add this line if you want to avoid checking in source code from Swift Package Manager dependencies.
 # Packages/
 .build/
+Package.resolved
```

Comment:
Since a new `Package.resolved` is created with each build, I'm adding this to  .gitignore